### PR TITLE
Improve UEFI Install Guide

### PR DESCRIPTION
**Pull request by andponlin**
For systems that require a UEFI boot process, this commit will improve the guidance for those users.

### CODE REVIEW

File: content/guides/uefi_booting.md [hunk 1]
```diff
@@ -1,59 +1,48 @@
...
+   3. Choose _Begin_
+   4. Once the installation is complete, choose _Quit_ to close the Installer.
+9. Shutdown the system.
+10. Remove the boot media; for example the USB key.
+11. Restart the system.
 
 ### Advanced Install
 
-To have Haiku on a system shared with multiple EFI operating systems, you can use the more advanced layout.
+To have Haiku on a system shared with multiple EFI operating systems, you can use the more advanced layout of the `EFIBOOT` partition;
```
Comment (humdingerb) [inline content/guides/uefi_booting.md:45]:
```suggestion
To have Haiku on a system shared with multiple EFI operating systems, you can use this more advanced layout on the `EFIBOOT` partition:
```

**Review by humdingerb — COMMENTED** [review]
If you want to, you can use e.g. `<span class="button">`Try Haiku`</span>` for buttons. There are also the classes menu, path, key.
`<pre class="terminal">`some terminal commands`</pre>` should css render a Terminal box.

**Comment by andponlin** [comment]
> If you want to, you can use...should css render a Terminal box.

Thanks Hundinger; I will stick with the tripple-quote for code blocks but I have added those other classes and applied your great suggestions.

**Comment by humdingerb** [comment]
Very good, let's merge!

**Pull request merged**